lib/utils/unbox.js

```diff
@@ -16,7 +16,7 @@ function checkDestination(destination) {
     var contents = fs.readdirSync(destination);
     if (contents.length) {
       var err = "Something already exists at the destination. " +
-                "Please unbox in an empty folder. " +
+                "Please run `truffle init` or `truffle unbox` in an empty folder. " +
```

Comment:
Also fixed this message, which has been the source of understandable bewilderment. People see this when they `truffle init`. Talking about boxes here is a little confusing for them.